src/markov/ddp.jl

```diff
@@ -589,7 +589,7 @@ s_wise_max!(vals::AbstractMatrix, out::AbstractVector) = (println("calling this
 
 """
 Populate `out` with  `max_a vals(s, a)`,  where `vals` is represented as a
-`AbstractMatrix` of size `(num_states, num_actions)`.
+`AbstractMatrix` of size `(num_states, num_actions).
```

Comment:
Same comment about backtick here

Reply:
yes, my bad. I will improve the formatting.